FeedStoreChallenge/Infrastructure/CoreData/CoreDataFeedStore.swift

```diff
@@ -29,14 +29,46 @@ public final class CoreDataFeedStore: FeedStore {
 	}
 
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		let context = context
+		context.perform {
```

Comment:
Suggestion:
Another way of capturing the `context` inside the closure without a variable is to use a capture list as shown below:

```suggestion
		context.perform { [context] in
```

It achieves the same result with one less line of code.